housecanary/excel/__init__.py

```diff
@@ -166,6 +216,15 @@ def _make_report_request(client, endpoint, address, zipcode, report_type):
     return {'success': True, 'content': response.content}
 
 
+def _save_individual_file(workbook, files_path, addr):
+    if not os.path.exists(files_path):
+        os.makedirs(files_path)
+
+    file_path = os.path.join(files_path, '{}-{}.xlsx'.format(addr, time.time()))
```

Comment:
Is the address normalized before it gets here?  Addresses can contain characters that would cause this to raise an exception (e.g. `Main Street 1/20`).